src/generateKarate.py

```diff
@@ -34,11 +34,16 @@ def getKarateType(prop):
     """
     Given an OpenApi property object, returns a Karate fuzzy matcher
     """
+
     karateType = '#notnull'
 
+    if isinstance(prop, unicode):
```

Comment:
When I run this in Python 3.7, I get `NameError: name 'unicode' is not defined`

Reply:
@alxjns I really need to get on 3.7; let me figure out why I'm a version behind and then refactor.